api/docs/http_certificate_status.md

```diff
@@ -13,7 +13,7 @@ Find
 ----
 
     GET /puppet-ca/v1/certificate_status/:certname?environment=:environment
-    Accept: pson
+    Accept: text/pson
```

Comment:
Here's what I'd expect puppetserver's behavior to currently be for variations of the Accept header in the request, assuming an authorization rule is configured which allows access:

* `Accept: pson` - 200 OK response with `Content-Type: pson`
* `Accept: text/pson` - 200 OK response with `Content-Type: text/pson`
* `Accept: application/json` - 200 OK response with `Content-Type: application/json`
* No Accept header - 200 OK response with `Content-Type: application/json`
* `Accept: other` - 406 (Not Acceptable) response with no response payload

In each of the 200 OK cases, the response payload would be serialized as json, even for the ones which would reply with some form of a "pson" Content-Type.